src/components/DatasetEditor/EditableTitleText.js

```diff
@@ -67,6 +32,7 @@ export default ({ value, onChange }) => {
       InputProps={{
         inputProps: { style: { color: "#000" } },
       }}
+      onBlur={handleBlue}
```

Comment:
minor typo: `handleBlue` -> `handleBlur`. fwiw I would avoid the word "handle" which can imply state. `onBlur` is fine for the name of a function (even though it's also the name of the prop).

Reply:
yes, makes sense